app/views/content/early-years-teaching-training.md

```diff
@@ -96,12 +80,34 @@ Bursaries are available, including:
 * £4,000 if you have a 2:1 degree
 * £2,000 if you have a 2:2 degree
 
+You are not eligible for a bursary if you have a third class degree.
+
+You do not need to do anything to access this funding. You can [learn more about early years initial teacher training funding](https://www.gov.uk/guidance/early-years-initial-teacher-training-2023-to-2024-funding-guidance).
+
+### Postgraduate employment-based training
+This is a route for graduates working in an early years setting who want to achieve early years teacher status.    
+
 $EY-graduate-employment$
 
-This one-year part-time route is for graduates working in an early years setting who need further training to meet the [early years teachers' standards](https://www.gov.uk/government/publications/early-years-teachers-standards).
+Up to £14,000 of funding is available. This would cover course fees of up to £7,000, plus £7,000 for your employer, to help cover their extra costs while you train. 
+
+You do not need to do anything to access this funding. You can [learn more about early years inital teacher training funding](https://www.gov.uk/guidance/early-years-initial-teacher-training-2023-to-2024-funding-guidance).
 
-Funding of up to £14,000 is available. This would cover course fees of up to £7,000, plus £7,000 for your employer, to help cover their extra costs while you train.
+### Undergraduate training
+This route is for people who want to study a full-time bachelor’s degree in an early-childhood related subject leading to EYTS.
+
+$EY-undergraduate$
+
+Undergraduate tuition fee loans are available from [Student Finance England (SFE)](https://www.gov.uk/student-finance).
+
+### Postgraduate 'assessment only' training
+
+This self-funded route is ideal for graduates with early-years experience. If you already meet the [early years teachers’ standards](https://www.gov.uk/government/publications/early-years-teachers-standards) you can gain EYTS without the need for further training.
```

Comment:
Does 'early-years' need a hyphen here? This is the only place on the page (but I don't know if that is because of the grammatical necessity in this context!)